WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunctionBundle\Tests\Functional\Repository;
 
+use OpenOrchestra\ModelInterface\Repository\Configuration\FinderConfiguration;
+use OpenOrchestra\ModelInterface\Repository\Configuration\PaginateFinderConfiguration;
```

Comment:
There is no need to add model-interface to the composer.json, he is already required by cms-bundle.

In all the worflow bundle we are using multiple class of the cms-bundle so model-interface are just some more